include/range/v3/at.hpp

```diff
@@ -35,10 +36,15 @@ namespace ranges
             auto operator()(Rng &&rng, range_difference_type_t<Rng> n) const ->
                 decltype(begin(rng)[n])
             {
+                if (SizedRange<Rng>()) {
+                    RANGES_EXPECT(n < ranges::distance(rng));
```

Comment:
Fixed. I'll love this OCD of yours. I wonder if I might be able to have it someday as well.